src/iqm/qiskit_iqm/iqm_move_layout.py

```diff
@@ -34,11 +38,7 @@ class IQMMoveLayout(TrivialLayout):
     Note: This simple version of the mapper only works reliably with a single move qubit
     and resonator, and only if the circuit contains at least one move gate."""
```

Comment:
This docstring should be more specific about what is being done. What if no initial layout makes the circuit valid for the given target? What does "single move qubit" mean here?

Is this function only called from `transpile_to_IQM`?

Does it need an IQMBackend, or only an IQMTarget as input?

Reply:
Is `generate_initial_layout` even needed? As far as I can tell it's not called when you use `transpile`, or provide an explicit target to `transpile_to_IQM`.

Reply:
It is needed if you have move gates in the circuit already to make sure that they are mapped properly. This use case is not supported when using the qiskit transpiler.